terraform/backend-prod/main.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+  #  backend "s3" {
+  #    bucket         = "gdx-data-share-tfstate"
+  #    key            = "terraform.tfstate"
+  #    region         = "eu-west-2"
+  #    dynamodb_table = "gdx-data-share-lock"
+  #    encrypt        = true
+  #  }
```

Comment:
why comment?

Reply:
Is that because it will fail before the bootstrap has run?

Reply:
Yep, we need to run it commented, then uncomment and run, so after it has been completed I would like a PR with it uncommented basically